addon/components/lt-body.js

```diff
@@ -175,8 +177,56 @@ export default Component.extend({
   useVirtualScrollbar: false,
 
   /**
+   * Set this property to scroll to a specific px offset.
+   *
+   * This only works when `useVirtualScrollbar` is `true`, i.e. when you are
+   * using fixed headers / footers.
+   *
+   * @property scrollTo
+   * @type {Number}
+   * @default null
+   */
+  scrollTo: null,
+
+  /**
+   * Set this property to a `Row` to scroll that `Row` into view.
+   *
+   * This only works when `useVirtualScrollbar` is `true`, i.e. when you are
+   * using fixed headers / footers.
+   *
+   * @property showRow
+   * @type {Row}
+   * @default null
+   */
+  showRow: null,
```

Comment:
I think `scrollToRow` is a more fitting name here and is consistent with the other `scrollTo` property.